Initial State:  os.statvfs_result(f_bsize=4096, f_frsize=4096, f_blocks=4096, f_bfree=3045, 
                f_bavail=3045, f_files=128, f_ffree=123, f_favail=123, f_flag=4102, f_namemax=252)
stat(shake-med.txt) got unexpected exception after unlink: [Errno 5] Input/output error: 'shake-med.txt'
Root directory mtime not updated after unlink.
Final State:    os.statvfs_result(f_bsize=4096, f_frsize=4096, f_blocks=4096, f_bfree=3045, 
                f_bavail=3045, f_files=128, f_ffree=123, f_favail=123, f_flag=4102, f_namemax=252)
After removing shake-med.txt, root dir listing is: ['shake-short.txt', 'shake-long.txt', 'truncate_me']
Root directory listing after unlink appears to be correct.
Freed 0 blocks, but should have freed 10.
Elapsed time: 0.32550254301168025 seconds
